src/browser/nativeModule.ts

```diff
@@ -148,7 +149,7 @@ export const browserNativeModuleRNPurchases = {
     ensurePurchasesConfigured();
 
     if (isExpoGo()) {
-      throw new Error('Purchasing is not currently supported in Expo Go');
+      return await purchaseSimulatedPackage(packageIdentifier, presentedOfferingContext);
```

Comment:
Are we sure that at this point only simulated purchases can take place in Expo Go, right? I ask to double check that we can't purchase web products from Expo Go. 
It's a bit confusing to me. Too many combinations 😅

Reply:
Well, in Expo Go we use the purchases-js SDK (so this browser module), so you could potentially use an rc billing API key. However, the purchase method wouldn't work there, that's why we were failing here until now. We do have a check in purchases-js now that, if trying to use a rc billing API key it would fail early though: https://github.com/RevenueCat/purchases-js/blob/main/src/main.ts#L840-L845

Reply:
That makes sense! Thank you for the explanation!